packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,6 +2,82 @@
 
 ## Changelog
 
+### Apr 18, 2023
+
+New features:
+
+- Run and debug Teams bot and message extension applications with Dev Tunnel.
+  ![devtunnel](https://user-images.githubusercontent.com/11220663/230364699-ed108641-4196-4318-949b-17ff56a6593f.png)
+
+- `Validate Application` command can now catch publish errors and provide mitigation suggestions for the application package.
```

Comment:
"`Validate Application` now helps catch errors and warnings that would prevent a successful publish to Teams in addition to schema checks."